source/code-snippets/usage-examples/count.js

```diff
@@ -0,0 +1,50 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // Estimate the total number of documents in the collection
+    // and print out the count.
+    collection.estimatedDocumentCount({}, function(error, result) {
+      if (error) {
+        console.log("Error: " + error);
+      } else {
+        console.log(
+          "Estimated number of documents in the movies collection: " + result,
+        );
+      }
+    });
+
+    // Query for movies from Canada.
+    const query = { countries: "Canada" };
+
+    // Find the number of documents that match the specified
+    // query, (i.e. with "Canada" as a value in the "countries" field)
+    // and print out the count.
+    collection.countDocuments(query, {}, function(error, result) {
```

Comment:
Turns out that going forward the team has decided to stick with Promises for the usage examples, so we should probably switch these examples away from callbacks (tl;dr: js newcomers are more familiar with Promises than callbacks, and we're trying to keep things as approachable as possible to them)

As for the existing examples that use callbacks, I'll probably PR an edit that uses Promises instead sometime in the near future. Sorry if you've had conflicting signals on some of this stuff :disappointed:

Reply:
Curses! Works for me 😁